### PR TITLE
[2203] Add PS Cmdlets for VPN Admin API

### PR DESCRIPTION
**Pull request by iancaragol**
The main new addition is the Get-AzSVirtualNetworkGatewayConnection cmdlet which uses talks to NRP to list all VPN resources created on the stamp.

As part of the generation for this cmdlet, many other files were updated as well. The docs MD file for each of the other cmdlet has a new format.

Please let me know if I need to remove these updates.

Thanks!

### CODE REVIEW

File: src/Azs.Network.Admin/Azs.Network.Admin.psd1 [hunk 1]
```diff
@@ -0,0 +1,25 @@
+@{
+  GUID = '70926bce-bc6d-44d9-9c7c-04173e27f5c9'
```
Comment (bganapa) [inline src/Azs.Network.Admin/Azs.Network.Admin.psd1:2]: This file should not be checked in, it gets generated and modified with the metadata specified in the readme.md

**Review by bganapa — CHANGES_REQUESTED** [review]
Update module version and the changelog. code review checklist has pointers
https://github.com/Azure/azurestack-powershell/wiki/Code-Review-Check-List

File: src/Azs.Network.Admin/Azs.Network.Admin.psd1 (hunk 1 above)
Comment (iancaragol) [reply src/Azs.Network.Admin/Azs.Network.Admin.psd1]: Thanks I removed it

**Comment by iancaragol** [comment]
@TheOnlyWei It looks like the CI run failed on the Backup.Admin tests. This should be unrelated to my changes. Is it a known issue?

**Comment by TheOnlyWei** [comment]
> @TheOnlyWei It looks like the CI run failed on the Backup.Admin tests. This should be unrelated to my changes. Is it a known issue?

Yes, it is a known issue. Backup RP is attempting to revert their changes, not don't know about ETA. I can force merge your changes as long as you haven't made changes to the RP that is failing.

**Comment by iancaragol** [comment]
Hi @TheOnlyWei 

We would prefer to go the force merge route if that is OK with you. That way everything is checked in and ready to go. I have only made changes within the AzS.Network.Admin space

Thanks!

**Pull request merged**